guides/2020-11-09-installing-go-programs-directly/en.markdown

```diff
@@ -34,6 +34,10 @@ Currently, tool authors who want to provide installation instructions in their p
 
 {{{ step "go115_mkcert_get" }}}
 
+_Note: most `README.md` instructions provide a `{{{ .cmdgo.get }}}` command without specifying a version. This results
+in the latest version of a package being fetched. A specific version, `{{{.mkcert_version}}}`, is used here to ensure
+this guide remains reproducible._
```

Comment:
This comment would be outdated fairly quickly, though. Once Go 1.16 is out in a couple of months, I expect a significant portion of READMEs to start using versioned `go install` commands.

Perhaps phrase this like "most READMEs using `go get`".

Reply:
Good suggestion, thank you.